src/views/apply/components/ApplySections/SocialInformation.tsx

```diff
@@ -19,8 +19,20 @@ function SocialInformation({
 >) {
 	return (
 		<div>
-			<h3>Social Information (Optional)</h3>
+			<h3>Social Information (Optional Section)</h3>
```

Comment:
Let's take out `(Optional Section)` since the bolded text below already tells us that.

Reply:
Fixed in 76b73fe.